runescape-api/src/main/java/net/runelite/rs/api/RSWidget.java

```diff
@@ -274,21 +271,11 @@
 	@Override
 	void setOriginalWidth(int originalWidth);
 
-	@Import("paddingX")
-	@Override
-	int getPaddingX();
-
-	@Import("paddingX")
-	@Override
-	void setPaddingX(int paddingX);
+	@Import("xPitch")
+	int getXPitch();
```

Comment:
So i assume this was removed from exports because it should not be messed with?

Reply:
I removed it because:
1) It seems very useless
2) Nothing used it
3) The docs were wrong anyway